README.md

```diff
@@ -82,13 +82,21 @@ The current set of feature sources are the following:
 
 The published node labels encode a few pieces of information:
 
-- A "namespace" (e.g. `node.alpha.kubernetes-incubator.io/nfd`).
+- A "namespace" (e.g. `node.alpha.kubernetes-incubator.io`).
 - The version of this discovery code that wrote the label, according to
   `git describe --tags --dirty --always`.
 - The source for each label (e.g. `cpuid`).
 - The name of the discovered feature as it appears in the underlying
   source, (e.g. `AESNI` from cpuid).
 
+Feature label names adhere to the following pattern:
+```
+<namespace>/nfd-<source-name>-<feature-name>[.<sub-feature-name>]
+```
+The last component (i.e. `sub-feature-name`) is optional, and only used if
+a feature logically has sub-hierarchy, e.g. `sriov.present` and `sriov.capable`
```

Comment:
Also nit. its sriov.present and sriov.capable here and below in the table its sriov.capable and sriov.configured

Reply:
Woops, a my fat brain made a mistake when playing around with the possible names. Fixed in next version of the PR